src/depth.rs

```diff
@@ -0,0 +1,74 @@
+use std::env;
+
+/// Supported color depths
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Copy, Clone)]
+pub enum Depth {
+    /// 3-bit color
+    TTY = 3,
+    /// 4-bit color
+    Low = 4,
+    /// 8-bit color
+    Medium = 8,
+    /// 24-bit color (true color)
+    High = 24,
+}
+
+impl Depth {
+    /// Try to inferr the best supported color depth for current terminal.
```

Comment:
Last one left over :pray:
```suggestion
    /// Try to infer the best supported color depth for current terminal.
```

Reply:
I should've done a search and replace :facepalm: